db/seed_maker.rb

```diff
@@ -24,7 +24,10 @@ def setup_fake_user
         user.last_name = 'Admin'
         user.password = user.password_confirmation = initial_password
         user.confirmed_at = Time.now
-        user.roles = [admin, dnd_staff]
+        user_group = UserGroup.where(name: 'Fake Admins').first_or_create
+        all_ds_entity_group = AccessGroup.where(name: 'All Data Sources').first
+        AccessControl.create(role: admin, access_group: all_ds_entity_group, user_group: user_group).add(user)
+        AccessControl.create(role: dnd_staff, access_group: all_ds_entity_group, user_group: user_group).add(user)
```

Comment:
I'm getting `unknown attribute 'access_group' for AccessControl` here. SHould we be creating a Collection instead of an AccessGroup?

Reply:
Ok this is what I had to do to get set up locally with acls on my account. cc @awisema though you probably already figured it out :)
```

user.update(permission_context: "acls")

all_ds_collection = Collection.where(name: 'All Data Sources').first

AccessControl.create(role: admin, collection: all_ds_collection, user_group: user_group).add(user)
```

Reply:
Yep, must have missed that one when rebuilding.  Thanks!
the collection should probably be
```all_ds_collection = Collection.system_collection(:data_sources)```